src/mozmlops/templates/README.md

```diff
@@ -27,7 +27,23 @@ You should see an instruction to run a command line command that starts with `ou
 > [!TIP]
 >The `--offline True` command line argument tells the template flow to not track experiments on W&B.
 
-The Metaflow flow will run internal consistency checks and linting, and then provide a link to track the progress of the flow on our platform, see stderr and stdout, et cetera. 
+The Metaflow flow will run internal consistency checks and linting, and then provide a link to track the progress of the flow on our platform, see stderr and stdout, et cetera.
+
+It'll do so on the Outerbounds platform, in the perimeter that you have specified. Perimeters need to be proactively granted access to resources whose access groups are more limited than mozilla-confidential.
+
+If you want to run the flow _locally_ with _your_ gcloud credentials to test out functionality that depends on permissions that _you_ have but the _perimeter_ doesn't, there are two steps to do that:
+
+1. (You only should ever have to do this once): Add an empty config set to your `.metaflowconfig` directory. This command will do it for you:
+
+```
+$echo '{}' > ~/.metaflowconfig/config_local.json
+```
+
+2. Run the command to kick off the job with `METAFLOW_PROFILE=local` on the front. For example:
+
+```commandline
+$METAFLOW_PROFILE=local python template_flow.py run --offline True
+```
```

Comment:
Would it be more clear if we add this part in a subsection?

Additionally, when I started reading `Running the template Metaflow flow` section, I didn't know whether my flow will run locally or on the cloud until I reached here. May be we can:
a) either specify in the heading that the following instructions are for the flow to run on cloud and add single subsection for running flows locally, OR
b) Let the section as it is and introduce 2 subsections within, one for running flows on cloud and one for running flows locally

However, this is just a suggestion as the documentation style is a matter of personal taste :)

Reply:
Ahh. I just realized that the title of the README is **Training and Deploying Models in The Cloud**. Then, `1` seems like the only option.

Reply:
That is an excellent callout. Thank you @abhi-agg ! I agree that a specific heading on this section is in order—possibly with some indication of _why_ you might want to do this (since the "point" of this template is to allow people to _not_ have to run locally). I'll make some updates today and push another commit.

Reply:
Update pushed!

Reply:
Thanks for making this change @chelseatroy 👍🏾